src/extractors/custom/weekly.ascii.jp/index.js

```diff
@@ -0,0 +1,31 @@
+export const WeeklyAsciiJpExtractor = {
+  domain: 'weekly.ascii.jp',
+
+  title: {
+    selectors: ['h1[itemprop]'],
+  },
+
+  author: {
+    selectors: ['p.author'],
+  },
+
+  date_published: {
+    selectors: ['p.data'],
```

Comment:
Let's fetch the date from the `datePublished` `meta` property to avoid having to define a custom format

Reply:
Thank you.
I will use `date_published` if it is in `meta`.